{
  "comments": [
    {
      "key": {
        "uuid": "1a76adc5_800625cd",
        "filename": "kubernetes/pomba/charts/pomba-data-router/values.yaml",
        "patchSetId": 4
      },
      "lineNbr": 66,
      "author": {
        "id": 1157
      },
      "writtenOn": "2018-08-22T19:41:29Z",
      "side": 1,
      "message": "These ports may or may not be a problem.  If the pomba-data-router and the data-router instances could co-exist then I think there would be an internal port conflict because the data-router internal and external port is also 9502.\n\nI guess the only question I would have is whether it is possible / likely that the pomba-data-router instance would be running side-by-side with the data-router?\n\nThe external node port you\u0027ve specified is different so it would be 30249, which should be ok.",
      "range": {
        "startLine": 65,
        "startChar": 0,
        "endLine": 66,
        "endChar": 20
      },
      "revId": "dd45e430474d0de2a55e3a75e80de9c19d227a94",
      "serverId": "14b4e41f-c4e3-4fb9-9955-6a9b5656895a",
      "unresolved": false
    }
  ]
}